index.js

```diff
@@ -2,50 +2,114 @@
 //Is time to create things in the DOM!!. We want you to create a new 'div' Element and append it as a child of the element that has the 'id = a-1' (the element is in index.html file)
 // all right our new element needs style. For that, assign to the new element the className 'square'!. Each time the user clicks on the new element it has to respond changing the displayed text 'x' to 'o'
 
-
-
-
 //resolve // QUESTION 1 here
-
-
-
+let divQ1 = document.querySelector("#a-1"); //accessing div element from html
+
+function createDiv() {
+  // this function creates a new div element, and a button with X text.
+  let newDiv = document.createElement("div");
+  let buttonX = document.createElement("button");
+  let buttonText = document.createTextNode(" X ");
```

Comment:
in line 12 .createTextNode(" X "); there is space next to X 
in  line 22 (event.target.textContent == "X")  there is not. thats why it doesn't change from X to O when first clicked